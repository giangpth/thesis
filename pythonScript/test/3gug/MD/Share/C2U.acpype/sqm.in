Run semi-empirical minimization
 &qmmm
    qm_theory='AM1', grms_tol=0.0005,
 scfconv=1.d-10, ndiis_attempts=700,   qmcharge=-1,
 /
   8   O7B        6.4560       -3.6590       19.9230 
   6    C7        6.7880       -3.6740       18.7030 
   8   O7A        7.4900       -2.7920       18.1590 
   6    C6        6.2510       -4.7500       17.8210 
   6    C1        6.4850       -4.7580       16.4470 
   6    C5        5.4920       -5.7510       18.4060 
   8    O5        5.2640       -5.7500       19.7300 
   6    C4        4.9150       -6.7590       17.6560 
  17   Cl4        3.9090       -7.9600       18.5180 
   6    C3        5.1270       -6.7990       16.2860 
   6    C2        5.9090       -5.8050       15.7020 
  17   Cl2        6.1860       -5.8980       13.9480 
   1    H1        7.0830       -3.9930       15.9740 
   1    H3        4.6950       -7.5850       15.6840 
   1    H5        5.5700       -4.9220       20.1070 

