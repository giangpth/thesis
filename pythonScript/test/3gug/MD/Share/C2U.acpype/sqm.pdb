ATOM      1  O7B C2U   351       6.615  -3.533  19.892  1.00  0.00           O
ATOM      2  C7  C2U   351       6.830  -3.592  18.642  1.00  0.00           C
ATOM      3  O7A C2U   351       7.525  -2.750  18.015  1.00  0.00           O
ATOM      4  C6  C2U   351       6.215  -4.740  17.854  1.00  0.00           C
ATOM      5  C1  C2U   351       6.458  -4.792  16.484  1.00  0.00           C
ATOM      6  C5  C2U   351       5.419  -5.741  18.456  1.00  0.00           C
ATOM      7  O5  C2U   351       5.128  -5.774  19.787  1.00  0.00           O
ATOM      8  C4  C2U   351       4.883  -6.777  17.657  1.00  0.00           C
ATOM      9 Cl4  C2U   351       3.913  -8.016  18.323  1.00  0.00          Cl
ATOM     10  C3  C2U   351       5.134  -6.816  16.284  1.00  0.00           C
ATOM     11  C2  C2U   351       5.921  -5.822  15.707  1.00  0.00           C
ATOM     12 Cl2  C2U   351       6.228  -5.872  14.025  1.00  0.00          Cl
ATOM     13  H1  C2U   351       7.083  -3.998  16.037  1.00  0.00           H
ATOM     14  H3  C2U   351       4.713  -7.625  15.670  1.00  0.00           H
ATOM     15  H5  C2U   351       5.555  -5.006  20.232  1.00  0.00           H
