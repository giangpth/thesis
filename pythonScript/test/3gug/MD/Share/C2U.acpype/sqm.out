            --------------------------------------------------------
                             AMBER SQM VERSION 19
 
                                     By
              Ross C. Walker, Michael F. Crowley, Scott Brozell,
                         Tim Giese, Andreas W. Goetz,
                        Tai-Sung Lee and David A. Case
 
            --------------------------------------------------------
 

--------------------------------------------------------------------------------
  QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 32

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: O : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: C : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: Cl: M.J.S.DEWAR et al. THEOCHEM, 180, 1, (1988)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)

| QMMM: *** SCF convergence criteria ***
| QMMM: Energy change                :  0.1D-09 kcal/mol
| QMMM: Error matrix |FP-PF|         :  0.1D+00 au
| QMMM: Density matrix change        :  0.5D-06
| QMMM: Maximum number of SCF cycles :     1000

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is in use.
| QMMM:
| QMMM: Timing diagonalization routines:
| QMMM:                              norbs =       51
| QMMM:    diag iterations used for timing =       15
| QMMM:
| QMMM:              Internal diag routine = 0.010617 seconds
| QMMM:                 Dspev diag routine = 0.028728 seconds
| QMMM:                Dspevd diag routine = 0.013426 seconds
| QMMM:                Dspevx diag routine = 0.058534 seconds
| QMMM:                 Dsyev diag routine = 0.020853 seconds
| QMMM:                Dsyevd diag routine = 0.013248 seconds
| QMMM:                Dsyevr diag routine = 0.016412 seconds
| QMMM:
| QMMM:                Pseudo diag routine = 0.002977 seconds
| QMMM:
| QMMM: Using internal diagonalization routine (diag_routine=1).

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      O        6.4560   -3.6590   19.9230
  QMMM:     2        2      C        6.7880   -3.6740   18.7030
  QMMM:     3        3      O        7.4900   -2.7920   18.1590
  QMMM:     4        4      C        6.2510   -4.7500   17.8210
  QMMM:     5        5      C        6.4850   -4.7580   16.4470
  QMMM:     6        6      C        5.4920   -5.7510   18.4060
  QMMM:     7        7      O        5.2640   -5.7500   19.7300
  QMMM:     8        8      C        4.9150   -6.7590   17.6560
  QMMM:     9        9      Cl       3.9090   -7.9600   18.5180
  QMMM:    10       10      C        5.1270   -6.7990   16.2860
  QMMM:    11       11      C        5.9090   -5.8050   15.7020
  QMMM:    12       12      Cl       6.1860   -5.8980   13.9480
  QMMM:    13       13      H        7.0830   -3.9930   15.9740
  QMMM:    14       14      H        4.6950   -7.5850   15.6840
  QMMM:    15       15      H        5.5700   -4.9220   20.1070

--------------------------------------------------------------------------------
  RESULTS
--------------------------------------------------------------------------------

      iter         sqm energy              rms gradient
      ----    -------------------    -----------------------
xmin    10     -156.3485 kcal/mol        0.1085 kcal/(mol*A)
xmin    20     -156.3540 kcal/mol        0.0477 kcal/(mol*A)
xmin    30     -156.2705 kcal/mol        0.7302 kcal/(mol*A)
xmin    40     -156.3366 kcal/mol        0.3704 kcal/(mol*A)
xmin    50     -156.3544 kcal/mol        0.0430 kcal/(mol*A)
xmin    60     -156.3554 kcal/mol        0.0783 kcal/(mol*A)
xmin    70     -156.3557 kcal/mol        0.0084 kcal/(mol*A)
  ... geometry converged !
 
 Final MO eigenvalues (eV):
 -37.1562  -35.0605  -34.6352  -33.7306  -33.2369  -29.8937  -28.2072  -26.8416
 -21.2817  -20.0803  -17.2027  -15.4128  -14.6221  -13.7836  -12.3140  -12.2955
 -11.9678  -11.6362  -10.9894  -10.8074  -10.3770  -10.3441  -10.0079   -9.3358
  -9.0796   -8.8429   -8.4454   -6.4934   -5.9012   -5.6800   -5.5786   -5.4872
   3.2562    3.4026    4.1174    4.5778    5.6291    6.3042    6.8912    7.2752
   7.3167    7.4992    7.7398    7.8279    7.9862    8.3241    8.7130    8.8787
   9.9135   11.7545   12.5518

 Heat of formation   =       -156.35566274 kcal/mol  (       -6.78009031 eV)

 Total SCF energy    =     -61072.16797666 kcal/mol  (    -2648.28793099 eV)
 Electronic energy   =    -251636.39817111 kcal/mol  (   -10911.77304415 eV)
 Core-core repulsion =     190564.23019445 kcal/mol  (     8263.48511315 eV)
 
    Atomic Charges for Step       1 :
  Atom    Element       Mulliken Charge
     1      O                 -0.581
     2      C                  0.369
     3      O                 -0.533
     4      C                 -0.189
     5      C                 -0.056
     6      C                  0.154
     7      O                 -0.286
     8      C                 -0.131
     9      Cl                -0.048
    10      C                 -0.105
    11      C                 -0.123
    12      Cl                -0.069
    13      H                  0.173
    14      H                  0.136
    15      H                  0.289
 Total Mulliken Charge =      -1.000
 
                  X        Y        Z     TOTAL  
  QM DIPOLE   -31.631   19.154  -89.010   96.386
 
 Final Structure

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      O        6.6154   -3.5334   19.8917
  QMMM:     2        2      C        6.8296   -3.5922   18.6417
  QMMM:     3        3      O        7.5249   -2.7499   18.0153
  QMMM:     4        4      C        6.2155   -4.7401   17.8543
  QMMM:     5        5      C        6.4581   -4.7919   16.4837
  QMMM:     6        6      C        5.4192   -5.7407   18.4563
  QMMM:     7        7      O        5.1276   -5.7735   19.7866
  QMMM:     8        8      C        4.8828   -6.7767   17.6569
  QMMM:     9        9      Cl       3.9127   -8.0162   18.3227
  QMMM:    10       10      C        5.1342   -6.8162   16.2843
  QMMM:    11       11      C        5.9211   -5.8221   15.7068
  QMMM:    12       12      Cl       6.2279   -5.8722   14.0249
  QMMM:    13       13      H        7.0834   -3.9984   16.0373
  QMMM:    14       14      H        4.7132   -7.6255   15.6699
  QMMM:    15       15      H        5.5546   -5.0060   20.2315

           --------- Calculation Completed ----------

